rpc/core/mempool.go

```diff
@@ -98,6 +143,50 @@ func BroadcastTxAsync(tx types.Tx) (*ctypes.ResultBroadcastTx, error) {
 // | Parameter | Type | Default | Required | Description     |
 // |-----------+------+---------+----------+-----------------|
 // | tx        | Tx   | nil     | true     | The transaction |
+//
+// To send a UTF8 string byte array, you simply quote the value of the tx parameter on the GET request:
```

Comment:
Can we deduplicate somehow? Will slate let us link to a previous methods description or to some comment documentation?

Reply:
I think it's totally fine to duplicate (until we have better RPC docs / switch to swagger https://github.com/tendermint/tendermint/issues/2419) and not force a person to go somewhere else.

Reply:
Actually, looks like we already had this info in "Using Tendermint". Replaced with the link